### ISSUE TITLE
Support named day of week

### ISSUE THREAD

**Issue by JASchilz** [kipe/pycron#12]
The [crontab man page](https://linux.die.net/man/5/crontab) specifies support for named days of the week, in place of numbered days of the week. See for example:
```
5 4 * * sun     echo "run at 5 after 4 every sunday"
```

The man page doesn't enumerate these day names, but from documentation elsewhere it looks like they are all the first three letters of a day's name (MON, THU, etc.), and case insensitive.

I propose that support be added for these named days of the week in the cron-like string.

If you concur, I could leave it to you, or work on a pull request. Looking at your code, my solution would replace your `int(s)` casting with a new `to_int(s)` method that could convert number strings `"3" ~> 3` as well as day names `"WED" ~> 3`. Additional tests would be easy to add.

**Comment by kipe** [comment]
This would be an awesome addition, why haven't I thought of this! :astonished: 

The solution you suggest sounds logical, the code should be very simple, something like:
```python
def to_int(s):
    try:
        return int(s)
    except ValueError:
        pass

    if s.lower() in ['mon', 'monday']:
        return 1
    # other days
    # If a day isn't found, raise ValueError
    raise ValueError('Failed to parse day string to integer')
```

A pull request is of course welcome, but I'll be sure to implement this once my vacation starts, if you haven't had time to implement it by then :smile:

**Comment by JASchilz** [comment]
👍 I'll probably put in a pull request, but if you have an opportunity then don't wait on me. (:

**Comment by kipe** [comment]
So the PR is now open for comments... I perhaps did it in a more complicated way than initially planned, so comments are more than welcome :laughing:

**Comment by kipe** [comment]
Closed via #13
